### PR TITLE
Updated some codes

### PR DESCRIPTION
**Pull request by DhanushAdithya**
- Updated styles in the index.css
- Added favicon to show the icon on the address bar of browsers
- Made little adjustments in the index.js

### CODE REVIEW

File: css/index.css [hunk 1]
```diff
@@ -3,213 +3,182 @@
 body {
   font-family: Roboto;
   font-weight: 400;
-
   margin: auto;
-
   background-color: #fafafa;
   color: white;
-
```
Comment (DhanushAdithya) [inline css/index.css]: Removed the unwanted whitespaces from the index.css file

File: css/index.css [hunk 2]
```diff
@@ -3,213 +3,182 @@
...
-.btnmenu {
-  float: right;
-  visibility: hidden;
-
-  padding: 2px 18px;
-
-  margin-top: 2px;
-  margin-right: 8px;
-
-  font-family: "Roboto";
```
Comment (DhanushAdithya) [inline css/index.css]: Interchanged the position according to the index.html file

File: css/index.css [hunk 3]
```diff
@@ -3,213 +3,182 @@
...
-
 .header {
-  width: 100% auto;
-
-  padding: 8px;
-
-  background-image: linear-gradient(
-    45deg, #0f6292, #12c055
-  );
+  display: flex;
```
Comment (DhanushAdithya) [inline css/index.css]: Replaced everything with flex as it is more responsive than native CSS

File: css/index.css [hunk 4]
```diff
@@ -3,213 +3,182 @@
...
 .container {
+  display: block;
   color: black;
-
   width: 100%;
-  height: 100vh;
-
-  margin-top: 56px;
-
-  overflow-y: scroll;
```
Comment (DhanushAdithya) [inline css/index.css]: Removed overflow-y property because of the container grows with the content you add later into it

File: css/index.css [hunk 5]
```diff
@@ -3,213 +3,182 @@
...
+  display: flex;
+  justify-content: center;
+  align-items: center;
   width: 100%;
   height: 110px;
-
-  background-image: linear-gradient(
-    45deg, rgb(16, 125, 175), rgb(30, 175, 233)
-  );
+  background-image: linear-gradient(45deg, rgb(16, 125, 175), rgb(30, 175, 233));
```
Comment (DhanushAdithya) [inline css/index.css]: I didn't even touch this thing.
I think its the work of the prettier in VS Code

File: css/index.css [hunk 6]
```diff
@@ -3,213 +3,182 @@
...
   color: white;
-
-  position: relative;
   overflow: hidden;
 }
 
-.presentation > .p {
-  position: absolute;
-  margin: auto;
```
Comment (DhanushAdithya) [inline css/index.css]: Removed everything and replaced with flex

File: index.html [hunk 7]
```diff
@@ -1,69 +1,78 @@
...
 
-    <link rel="stylesheet" href="css/index.css" />
+<head>
+  <title>Doweb</title>
 
-    <meta charset="UTF-8" />
-    <meta name="theme-color" content="#db5945" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="css/index.css" />
+  <link rel="shortcut icon" href="./img/icon.png" type="image/x-icon">
```
Comment (DhanushAdithya) [inline index.html]: Added favicon which will appear in the address bar when you open dob6458.github.io after you merge this pull request

File: index.html [hunk 8]
```diff
@@ -1,69 +1,78 @@
...
         <a href="https://discord.gg/x8Pr7fq" class="item">Rejoin us on Discord</a>
         <a href="mailto:dob6458@outlook.com" class="item">Contact us</a>
         <a href="https://github.com/Dob6458" class="item">Github</a>
       </div>
-      <button class="btnmenu">
-        <i class="material-icons icon-menu">
-          menu
-        </i>
-      </button>
+      <div class="btn-menu">
```
Comment (DhanushAdithya) [inline index.html]: Added another div for convenience

File: js/index.js [hunk 9]
```diff
@@ -43,17 +43,20 @@ $(document).ready(async function() {
 
     // Create "this.min" and "this.max" for the height of the header
     this.min = 40;
-    this.max = 154;
+    this.max = 205;
```
Comment (DhanushAdithya) [inline js/index.js]: Adjusted this.max value

File: js/index.js [hunk 10]
```diff
@@ -43,17 +43,20 @@ $(document).ready(async function() {
...
 
-    this.bool = function(bool) {
+    this.bool = function (bool) {
       // If "bool" variable is 'true'
       if (bool === true) {
         // Open the menu bar
         $(this.id).css("height", this.max);
+        $(this.id).css("align-items", "flex-start");
```
Comment (DhanushAdithya) [inline js/index.js]: I hope you know what these 2 lines will do with the code

**Comment by Dob6458** [comment]
Thank you for corrections 😉

**Comment by DhanushAdithya** [comment]
You're Welcome

**Pull request merged**